data/mods/Aftershock/mobs/uplifted_monsters.json

```diff
@@ -103,6 +103,7 @@
     "dodge": 4,
     "vision_day": 30,
     "death_drops": "xl_uplift_death_drop",
+    "path_settings": { "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true, "max_dist": 10 },
     "harvest": "demihuman_large_fur",
     "special_attacks": [ [ "scratch", 20 ] ],
     "zombify_into": "mon_uplifted_ape_zed",
```

Comment:
The line below this has two copies of "PATH_AVOID_DANGER_1", BTW.

Reply:
Thanks for catching that!